api/src/test/java/org/apache/iceberg/types/TestConversions.java

```diff
@@ -93,15 +93,26 @@ public void testByteBufferConversions() {
     assertThat(Literal.of(10000L).to(TimeType.get()).toByteBuffer().array())
         .isEqualTo(new byte[] {16, 39, 0, 0, 0, 0, 0, 0});
 
-    // timestamps are stored as microseconds from 1970-01-01 00:00:00.000000 in an 8-byte
+    // timestamps are stored as micro|nanoseconds from 1970-01-01 00:00:00 in an
+    // 8-byte
```

Comment:
Can you fix the line wrapping in this comment?

Reply:
Done.